src/perception/lidar_pipeline/lidar_pipeline/node_lidar_detector.py

```diff
@@ -173,7 +173,7 @@ def callback(self, msg: PointCloud2):
         # Step 1
 
         # Convert PointCloud2 message from LiDAR sensor to numpy array
-        dtype_list: list = fields_to_dtype(msg.fields, msg.point_step)  # x, y, z, intensity, ring
+        dtype_list: list = fields_to_dtype(msg.fields, msg.point_step)  # x, y, z, intensity, ringr
```

Comment:
Not critical, in the future, double check what files and lines are being staged for the commit and only select stage file on only the ones the feature is about.
(ie. not this lidar file)